src/test/java/org/jabref/logic/integrity/ASCIICharacterCheckerTest.java

```diff
@@ -0,0 +1,24 @@
+package org.jabref.logic.integrity;
+
+import org.jabref.model.entry.field.StandardField;
+
+import org.junit.jupiter.api.Test;
+
+public class ASCIICharacterCheckerTest {
+
+    @Test
+    void fieldAcceptsASCIICharacters() {
```

Comment:
```suggestion
    void fieldAcceptsAsciiCharacters() {
```

Even not conform with current JabRef classes, please use defined camel case - see https://google.github.io/styleguide/javaguide.html#s5.3-camel-case for details.